### DISCUSSION TITLE
重复、虚假大学 集中反馈帖

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#90] [Announcements]
Deprecated: 请转到 [问卷数据](https://github.com/CollegesChat/university-information/tree/generated) 或  [colleges.chat 网站](https://colleges.chat) 以查看更多更全的数据。如有相关反馈，请到 [issue 页面](https://github.com/CollegesChat/university-information/issues) 新建 issue。

---

> 重复大学，如 **同济大学** 与 **上海同济大学**，可进行合并（不同校区不合并）。
> 虚假大学，如 **家里蹲大学** 或 **喵内噶大学**，可进行删除。
> 
> Note: **简繁已处理**
> 
> 由于一条一条回复会有点多…，如果收到反馈我们会 react 一个 ♥。
> 如果回复后，要补充大学，建议再回复，不建议编辑，因为编辑不会有通知。
> 
> ---
> 
> 建议数据提交格式： `官方名称,重复名称1,重复名称2,...` 。
> 每个名称间使用半角逗号隔开，方便处理。
> By @renbaoshuo

**Comment by slanterns** [comment]
中國美術學院 - 中国美术学院
復旦大學 - 复旦大学

虚假大学：仲恺说唱职业技术学院

**Reply by YanWQ-monad** [reply]
简繁已处理，忘记说了，不好意思

**Comment by VCStardust** [comment]
重复：
哈尔滨工业大学(威海) - 哈尔滨工业大学（威海）
哈尔滨工业大学深圳 - 哈尔滨工业大学（深圳）
山东大学（威海校区） - 山东大学（威海）
山东大学（青岛校区） - 山东大学（青岛）
郑州大学北校区 - 郑州大学（北校区）
福建幼儿师范高等专科学院 - 福建幼儿师范高等专科学校
华南理工大学大学城校区 - 华南理工大学（大学城校区）

其它：
`中山大学（珠海校区，括号删掉）` ？

**Reply by YanWQ-monad** [reply]
已经暴力把括号干掉了（虽然有的会看起来很奇怪）

**Reply by nexplorer-3e** [reply]
以及 his 文档里是属于东北地区的
虽然和 hit 放一块~~也好找~~

**Reply by YanWQ-monad** [reply]
his 全称是什么……（Google 告诉我是 H.I.S.Co.，Ltd.）
再者，网站已经干掉分区，变成省份了

**Comment by slanterns** [comment]
复旦大学,復旦大學,上海复旦大学
虚假大学：麻省理工（建议把那份问卷干掉）

**Comment by Xeonacid** [comment]
哈尔滨工业大学,哈尔滨工业大学二校区,哈尔滨工业大学二校区校本部

**Comment by Archeb** [comment]
北京师范大学珠海**分校**已停止招生，建议标明；目前招生的是北京师范大学珠海**校区**，还有名字听起来很像的北师大浸会联合国际学院。同时以上三个学校都坐落于广东珠海。
（但因为分校与校区共用校园，故生活、宿舍方面的数据仍有参考价值）

**Comment by lilongsky** [comment]
虚假大学：[国立风府大学](https://colleges.chat/universities/guo-li-feng-fu-da-xue/)

**Comment by jjq2004** [comment]
重复:
大连理工大学,大连理工大学本部,大连理工大学主校区
大连理工大学开发区校区,大连理工大学软件学院

**Comment by QuartzKun** [comment]
重复：成都信息工程大学,成都信息工程大学航空港校区

**Comment by PetrichorA** [comment]
重复：「香港城市大学」和「City University of Hong Kong」

https://colleges.chat/universities/city-university-of-hong-kong/
https://colleges.chat/universities/xiang-gang-cheng-shi-da-xue/

**Comment by nicholascw** [comment]
University of California San Diego == University of California, San Diego
University of Illinois Urbana Champaign == University of Illinois at Urbana-Champaign

以及 龙城高级中学 这里怎么混进来个高中 ❓

**Comment by RoderickQiu** [comment]
看到广东处发现华南理工有重复性

**Comment by chenyumenb** [comment]
没看到广东工业大学揭阳分校

**Comment by wuduanq** [comment]
武汉大学 国立武汉大学 武汉大学信息学部重复，需合并
另 武汉大学国家网络安全学院和国家网络安全学院为同一单位，需合并

**Comment by jjq2004** [comment]
虚假大学 加里敦大学 哈哈大学

**Comment by Qnurye** [comment]
虚假帖：哈哈大学

**Comment by Chastinian** [comment]
虚假大学：下北泽会员制大学、下北泽野兽学院
建议把这俩虚假贴挂起来当吉祥物

**Reply by Chastinian** [reply]
这俩虚假帖疑似已被橄榄

**Comment by Chastinian** [comment]
重复大学：西交利物浦大学、西安交通利物浦大学
建议将“西安交通利物浦大学”贴并入“西交利物浦大学”贴

**Comment by Evyde** [comment]
吉林大学前卫南区,吉林大学中心校区/前卫南区,吉林大学前卫南区,吉林大学南前卫南区

**Comment by BakaJzon** [comment]
[哈尔滨工业大学深圳校区](https://colleges.chat/universities/ha-er-bin-gong-ye-da-xue-shen-zhen-xiao-qu/)
[哈尔滨工业大学深圳](https://colleges.chat/universities/ha-er-bin-gong-ye-da-xue-shen-zhen/)

**Comment by HUAWEIGI** [comment]
武夷学院 A12685
宁波职业技术学院 A13003
A15120 答非所问
A13981 乱填数据

**Comment by HUAWEIGI** [comment]
A14660胡言乱语

**Comment by m1dnightsong** [comment]
“深圳大学”与“！深圳大学”

**Comment by HUAWEIGI** [comment]
江西制造职业技术学院 A14862 胡乱作答！
丽江高等师范 A15120 胡乱作答！

**Comment by rainRb** [comment]
哈尔滨工业大学二校区 - 哈尔滨工业大学二小区

**Comment by HUAWEIGI** [comment]
重庆电力高等专科学 A20428 胡乱作答

**Comment by HUAWEIGI** [comment]
黑龙江工业学院  A20435 胡乱作答

**Comment by HUAWEIGI** [comment]
[南京邮电大学仙林校区，但是我住的宿舍是全校条件最好的青教，但是相对的最偏僻，离除了学科楼之外的地方都很远](https://cn.colleges.chat/universities/nan-jing-you-dian-da-xue-xian-lin-xiao-qu-dan-shi-wo-zhu-de-su-she-shi-quan-xiao-tiao-jian-zui-hao-de-qing-jiao-dan-shi-xiang-dui-de-zui-pian-pi-chi-chu-liao-xue-ke-lou-zhi-wai-de-di-fang-du-hen-yuan) 

这个学校名字太长了

**Reply by HUAWEIGI** [reply]
[南通大学 如无特殊，以下仅针对启秀校区包含医学系护理学院药学院，另本人以毕业四五年，可能有更新 (已归档)](https://cn.colleges.chat/archived/universities/nan-tong-da-xue-ru-wu-te-shu-yi-xia-jin-zhen-dui-qi-xiu-xiao-qu-bao-han-yi-xue-xi-hu-li-xue-yuan-yao-xue-yuan-ling-ben-ren-yi-bi-ye-si-wu-nian-ke-neng-you-geng-xin/)

**Reply by Rachel030219** [reply]
上面那个将在下一次数据更新时处理，下面这个考虑到已经归档了，就让它归档吧🙏

**Comment by HUAWEIGI** [comment]
南昌应用技术师范学院/江西科技师范大学理工学院
现用名 南昌应用技术师范学院

**Comment by HUAWEIGI** [comment]
山东第一医科大学山东省医学科学院  应该是 山东第一医科大学

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/archived/universities/jiang-xi-cai-jing-da-xue-mai-lu-xiao-qu-jiao-qiao-xiao-qu-du-zhu-guo/
希望能简化一下

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/tian-jin-ke-ji-da-xue-tai-da-xiao-qu-jiu-shi-bin-hai-xiao-qu/
全名应该是 ：天津科技大学滨海校区

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/tian-jin-ke-ji-da-xue-tai-da-xiao-qu-jiu-shi-bin-hai-xiao-qu/
全名应该是 ：天津科技大学滨海校区

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/nan-chang-jiao-tong-xue-yuan-yi-xia-hui-da-mo-ren-mo-xuan-hu-xiao-qu/
南昌交通学院墨轩湖校区

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/gan-nan-shi-fan-da-xue-shuo-shi-yan-jiu-sheng-zhong-shan-da-xue-xin-hua-xue-yuan-ben-ke/

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/guo-li-wu-yi-da-xue-wu-yi-da-xue/
武汉大学

**Comment by HUAWEIGI** [comment]
https://cn.colleges.chat/universities/zhong-guo-yan-dong-sheng-fo-shan-shi-shan-cheng-qu-fo-shan-ke-xue-ji-zhu-xue-yuan/
佛山科学技术学院

**Comment by HUAWEIGI** [comment]
东莞理工学院城市学院 已经改名为 东莞城市学院

**Comment by Kifranei** [comment]
麻烦把“哈尔滨职业技术学院哈尔滨职业技术大学”删掉吧，我重新填写了一份11月的问卷

**Comment by Young-Lord** [comment]
虚假大学：
https://colleges.chat/universities/miskatonic-university/
无意义填写：
https://colleges.chat/universities/gen/
错误填写校名：
https://colleges.chat/universities/kutuya2024-gmail-com/

**Comment by ghost** [comment]


**Reply by ghost** [reply]
https://colleges.chat/universities/da-lian-wai-guo-yu-da-xue/中A20163的回答属于胡言乱语

**Reply by soberup** [reply]
那是不是也得把A22670和A17480删了啊
贵校老师专门注册GitHub就为这事啊，吃相别太难看

**Comment by soberup** [comment]
嘉兴大学，嘉兴学院
浙江科技大学，浙江科技学院
二者已于2023年分别更名为嘉兴大学与浙江科技大学

**Comment by Mikachu2333** [comment]
潍坊医学院已更名为山东第二医科大学

**Comment by ahpxex** [comment]
https://colleges.chat/universities/123/
https://colleges.chat/universities/h-m-q-2702056270/
https://colleges.chat/universities/fa-de/
https://colleges.chat/universities/qia-sai-er-xue-yuan/
https://colleges.chat/universities/jin-sheng-jiang/
https://colleges.chat/universities/cun-jin/
https://colleges.chat/universities/xin-hua/
https://colleges.chat/universities/qing-hua/
胡乱填写、虚假大学、重复

**Reply by Mikachu2333** [reply]
哥们，你这回复错人吓了我一跳，我还以为我填的属于胡乱填写系列的，还在想谁在污蔑我（

刚刚邮箱收到了邮件但是没找明白，仔细一看才发现问题 🤣

**Reply by ahpxex** [reply]
好像没有回复错..... 这个 discussion 应该是自动把参与的人加入一个 mailing list 了，所以应该所有人都能收到邮件通知

**Reply by Mikachu2333** [reply]
？！原来是这样的吗，太离谱了……

**Reply by YanWQ-monad** [reply]
可以点右边的 Unsubscribe 来取消订阅。

**Reply by Mikachu2333** [reply]
> 可以点右边的 Unsubscribe 来取消订阅。

抓作者，问问啥时候集中整理一波啊，我发了问卷也提了改名但是没啥动静（

**Comment by KimmyXYC** [comment]
https://colleges.chat/universities/zhu-bo-nuo-ding-yi-da-xue/
https://colleges.chat/universities/zhu-bo-nuo-ding-yi-da-xue-university-of-nottingham-ningbo-china/
宁波诺丁汉大学重复

**Comment by Trut2110** [comment]
虚假大学：A27597，侵华大学

**Comment by pkx07** [comment]
厦门大学马来西亚分校,厦门大学马来西亚校区

**Reply by pkx07** [reply]
A27294将官方名称“厦门大学马来西亚分校”记录为“厦门大学马来西亚校区”。

**Comment by Counta** [comment]
北部湾大学-广西北部湾大学
重复
还有个学校名超长的：
北京科技大学全称是这个，但是我人在海淀本部校区，一般不写校区，还分其他昌平校区，管庄校区，这个一般写北京科技大学昌平/管庄校区

**Comment by XINGXIOVO** [comment]
作者大大我一直没有看到，广州华南商贸职业学院，不会是和广州华商职业学院混淆了吧。
